src/validations/Batch.js

```diff
@@ -64,6 +64,23 @@ export const BatchValidations = Yup.object({
   enrollment_type,
 });
 
+export const alterBatchValidations = Yup.object({
```

Comment:
instead of defining a separate validation definition, you can extend the existing validation and set validation rules dynamically.

Refer to [this code in Student validation](https://github.com/gitmedha/sis-frontend/blob/main/src/validations/Student.js#L88).

Reply:
I have removed the alterBatchValidations and made the batchValidations dynamics Please have a look over there, Thanks